packages/gestalt-design-tokens/build.js

```diff
@@ -150,6 +150,39 @@ StyleDictionary.registerFormat({
   },
 });
 
+StyleDictionary.registerFormat({
+  name: `constantLibrary-commonJS/flow`,
+  formatter({ dictionary, file }) {
+    const tokens = dictionary.allTokens
+      .map((token) => {
+        let value = JSON.stringify(token.value);
+        // the `dictionary` object now has `usesReference()` and
+        // `getReferences()` methods. `usesReference()` will return true if
+        // the value has a reference in it. `getReferences()` will return
+        // an array of references to the whole tokens so that you can access their
+        // names or any other attributes.
+        if (dictionary.usesReference(token.original.value)) {
+          // Note: make sure to use `token.original.value` because
+          // `token.value` is already resolved at this point.
+          const refs = dictionary.getReferences(token.original.value);
+          refs.forEach((ref) => {
+            value = value.replace(ref.value, () => `${ref.name}`);
```

Comment:
Let's make it simpler
```suggestion
            value = value.replace(ref.value, ref.name);
```

Reply:
DONE